### DISCUSSION TITLE
More keywords than the standard ones

### DISCUSSION THREAD

**Discussion by actopozipc** [actopozipc/German-Python-Interpreter#7] [Polls]
There are frameworks that mimic the behavior of keywords from other languages, such as goto. Could you imagine something like this to maximize the number of German keywords?

**Comment by KonkenBonken** [comment]
How would you even implement a `goto` keyword

**Reply by actopozipc** [reply]
There is a framework that imitates the behavior of goto and comefrom. One could insert the framework import before the interpreter translates the code and interprets it again 

Hope that makes sense

**Comment by KonkenBonken** [comment]
What other keywords were you planning to add, besides from `goto`?

**Reply by actopozipc** [reply]
Just comefrom right now
